documentation/content/Extra Customization/revealjs.md

```diff
@@ -0,0 +1,43 @@
+---
+author: Pablo Iranzo Gómez
+title: Pelican-Elegant and RevealMD presentations
+tags: pelican, foss, blog, python, github, blog-o-matic, linux, pandoc
+layout: post
+date: 2019-07-22 22:12:17 +0100
+comments: true
+category: tech
+description:
+---
```

Comment:
Please add slug variable. like `reveal-md-presentations-in-elegant`.

Reply:
done in next commit